relay-general/src/store/normalize.rs

```diff
@@ -1964,30 +1995,30 @@ mod tests {
 
     #[test]
     fn test_normalize_dist_none() {
-        let mut dist = None;
+        let mut dist = Annotated::default();
         normalize_dist(&mut dist);
-        assert_eq!(dist, None);
+        assert_eq!(dist.value(), None);
     }
 
     #[test]
     fn test_normalize_dist_empty() {
-        let mut dist = Some("".to_owned());
+        let mut dist = Annotated::new("".to_string());
         normalize_dist(&mut dist);
-        assert_eq!(dist, None);
+        assert_eq!(dist.value(), None);
     }
 
     #[test]
     fn test_normalize_dist_trim() {
-        let mut dist = Some(" foo  ".to_owned());
+        let mut dist = Annotated::new(" foo  ".to_string());
         normalize_dist(&mut dist);
-        assert_eq!(dist.unwrap(), "foo");
+        assert_eq!(dist.value(), Some(&"foo".to_string()));
     }
 
     #[test]
     fn test_normalize_dist_whitespace() {
-        let mut dist = Some(" ".to_owned());
+        let mut dist = Annotated::new(" ".to_owned());
         normalize_dist(&mut dist);
-        assert_eq!(dist.unwrap(), ""); // Not sure if this is what we want
+        assert_eq!(dist.value(), Some(&"".to_string())); // Not sure if this is what we want
```

Comment:
Normalization should be idempotent. If we make another normalization call, `dist` is then `None`. This situation must not happen -- we're running multiple layers of relays and debugging can be complicated.

Reply:
I'd also take the chance and make a decision with regard to that comment. If the resulting string after trimming is empty, `dist` should be `None`.

Reply:
agreed and fixed this